templates/common/_base/files/configure-ovs-network.yaml

```diff
@@ -490,6 +490,33 @@ contents:
       echo ""
     }
 
+    get_interface_from_ip_hint() {
+      local ip_hint=${NODE_IP:-""}
+      local extra_bridge="$1"
+      local iface=""
+      if [ "${ip_hint}" != "" ]; then
+      # check ipv4
+        if [ "${extra_bridge}" != "" ]; then
```

Comment:
Since this logic is copied from below, it would be nice to factor it out into a separate function.

Reply:
it looks familiar but it is actually different. Here i allow setting br-ex on any interface and not only the one with default route + adding grep on ip that. I assume if it was not bash i would create separate function but here it will not be less ugly but if you think it is worth it i will try to find out something

Reply:
Not that bash master to make it nicer to say the truth